pkgs/tools/misc/hdl-dump/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, fetchFromGitHub
+, upx
+}:
+
+let
+  version = "20202807";
+  pname = "hdl-dump";
+in stdenv.mkDerivation {
+  inherit pname version;
+
+  # playstation2 repo is outdated
```

Comment:
Please add a link which one.

Reply:
It is implict, but I will update it nonetheless.